openapiart/tests/pattern/pattern.yaml

```diff
@@ -8,7 +8,7 @@ components:
           x-field-pattern:
             format: ipv4
             default: 0.0.0.0
-            features: [count]
+            features: [count, random]
```

Comment:
Add one test without `random` feature and validate that `random` will not auto-populate 
```
        ipv4:
          x-field-pattern:
            format: ipv4
            default: 0.0.0.0
            features: [count, auto]
```

Reply:
can this be treated as an example , test is already present to check this pattern 
https://github.com/open-traffic-generator/openapiart/pull/467/files#diff-7df96b39be93404f05e016f7517caf3a05521ba323a82e6be906138fb72edcfcR87